R/pkg/NAMESPACE

```diff
@@ -281,6 +281,8 @@ exportMethods("%<=>%",
               "initcap",
               "input_file_name",
               "instr",
+              "isInf",
+              "isinf",
```

Comment:
I really don't understand why we have both. It usually has the ones matching to Scala side or R specific function. Otherwise, I don't think we should have both.

Reply:
Do you know if there's any consistency behind the different capitalization schemes? There's `format_number`, `isnan` and `isNotNull` here, for example. 

If not, how about we just go with `isInf` for now and if other aliases are needed in the future they can be added and discussed then?

Reply:
I think we shouldn't add other variants unless there's a clear reason. It sounds like we are adding this for no reason.

> how about we just go with isInf for now and if other aliases are needed in the future they can be added and discussed then?

Yea, please.

Reply:
I commented on this here https://github.com/apache/spark/pull/21482#discussion_r192577235
it's because Column capitalizes it differently from functions

Reply:
Ah, I got it now. I believe we should match it to one side though. I roughly remember we keep functions this_naming_style in functions[.py|.R|.scala], e.g.([SPARK-10621](https://issues.apache.org/jira/browse/SPARK-10621)). Shall we stick to `isinf` then?